docs/contributing/writing-instrumentation.md

```diff
@@ -402,6 +402,26 @@ All classes from the newly added bootstrap module will be loaded by the bootstra
 globally available within the JVM. **IMPORTANT: Note that you _cannot_ use any third-party libraries
 here, including the instrumented library - you can only use JDK and OpenTelemetry API classes.**
 
+### Common Modules
+
+When creating a common module shared among different instrumentations, the naming convention should
+include a version suffix that matches the version of the instrumented library specified in the
+common module's `build.gradle.kts`.
+
+For example, if the common module's Gradle file contains the following dependency:
+
+```kotlin
+dependencies {
+  compileOnly("org.yarpc.client:rest:5.0.0")
+}
+```
+
+Then the module should be named using the suffix `yarp-common-5.0`.
```

Comment:
Off the top of my head, I don't know how to phrase it...but if I had no familiarity and read this for the first time, I would wonder why the module isn't called `yarp-common-5.0.0`. Can we clarify that we use major/minor or lowest major/minor ?